src/bridge/ui_commands.rs

```diff
@@ -148,6 +149,11 @@ impl ParallelCommand {
             ParallelCommand::FileDrop(path) => {
                 nvim.command(format!("e {path}").as_str()).await.ok();
             }
+            ParallelCommand::SetBackground(background) => {
+                nvim.command(&format!("set background={}", background).as_str())
```

Comment:
As clippy already mentions, with `.as_str()`, the `&` is not required anymore.

```suggestion
                nvim.command(format!("set background={}", background).as_str())
```